sources/FILEIO

```diff
@@ -940,7 +940,7 @@ Copyright (c) 1981-1993, 1999, 2020-2021 by Venue & Xerox Corporation.
 
                     (* ;; "-----For any stream (here to not recompile everything)-----")
 
-                    (NIL POINTER)                            (* ; 
+                    (DEFAULTEXTERNALFORMAT POINTER)          (* ; 
```

Comment:
We inherited a notion that there is a default format for streams on a particular filedevice.  That has been specified indirectly, on an alist, so you could push (UNIX :UTF-8) or (DSK :XCCS) if that's how you want to make the distinction.  The function \EXTERNALFORMAT is called when a stream is opened, and it installs the default for that device, or a global default if nothing is specified for that particular device.  

I changed this so that there can be a stronger fall-back format for a particular device, stored in the device itself, although the more accessible alist is consulted first.

The immediate motivation was the fact that there was no separate default external format for the DISPLAY device, and it was defaulting to the system default, which had the wrong EOL setting for the display.  That only showed up when the display was being opened as a stream, e.g. in the original system definition of SEE (as opposed to the override of PRETTYFILEINDEX).  That could have been done by pushing something on the list, but again I thought it should be built in.